compute/admin_guide/install/system_requirements.adoc

```diff
@@ -162,7 +162,8 @@ Refer to the the Linux capabilities https://man7.org/linux/man-pages/man7/capabi
 
 [NOTE]
 ====
-The Prisma Cloud App-Embedded Defender requires CAP_SYS_PTRACE only. 
+* The Prisma Cloud App-Embedded Defender requires CAP_SYS_PTRACE only. 
```

Comment:
@Pubs-MV: I am moving this note [here](https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/1441/files#diff-3a2263c7dddc9c03b37ee429cfd3fa9b3835a3c1051a19717f34b6a87a02deb1:~:text=Radars%20%3E%20Settings*.-,NOTE,-%3A%20If%20you) with O'Neal release.